### DISCUSSION TITLE
cupture qr code to specific client

### DISCUSSION THREAD

**Discussion by danbinnun1** [open-wa/wa-automate-nodejs#2585] [Q&A] [answered]
I saw this solution to save qr code:
https://docs.openwa.dev/pages/The%20Client/launch-events/capture-qr.html#capturing-qr-code
But I have multiple clients so I need to know which qr code belongs to which client. How can I do it?

**Comment by smashah** [comment, marked as answer]
https://github.com/open-wa/wa-automate-nodejs/blob/000c92acd9f35db178b5dc7b7ef5a8c296654bc9/demo/index.ts#L27

**Reply by danbinnun1** [reply]
Thank you!
